### DISCUSSION TITLE
Trouble applying for Student Developer Pack

### DISCUSSION THREAD

**Discussion by arnavmeduri** [community/community#65847] [GitHub Education]
### Hi everyone,

Greetings,

I am having trouble applying for Github Student Education Discounts. I looked through this forum for any hints, tried multiple browsers including mobile browser, without success. 

I followed the steps outlined in the link [here](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)

After Step (7), "Click Continue, and then you will be prompted to upload proof of your academic status." - I am not getting an option to upload my school student ID (or a prompt to take a picture via phone). Instead, I get this cryptic message:

"Your school is a GitHub Campus partner. Please contact your school administrator to have the correct benefits applied to your GitHub account"

I have contacted my school administrator, and they say that the process should prompt to upload student ID.

What should I do now? Is it possible that the application "remembered" some of my settings from before, and is that the reason it is not prompting me again? If so, how can I reset all my submissions and start all over again?

Why am I not being prompted to upload student ID again? How can I reach gituhub support for this issue?

Much appreciate any guidance you can provide to help me get past this issue or point me to the right resources that can help me.

Have a great day,
Arnav

**Comment by arnavmeduri** [comment]
Can anyone provide guidace?

**Reply by SpurGrappler** [reply]
Were you able to get approved? I'm having this same issue, I did submit my student ID, I also cut and pasted some transcript screenshots as .JPGs so I could upload those as well (wouldn't allow for .PDFs). Hoping to figure this out

**Reply by arnavmeduri** [reply]
@SpurGrappler - I have not been able to. Don't know where to go from here. Pls let me know if you have success.
